.travis.yml

```diff
@@ -2,8 +2,10 @@ language: node_js
 before_install:
  - sudo apt-get install build-essential
  - npm i mocha -g
+ - npm i maboiteaspam/cozy-light#refactoring -g
 node_js:
-    - "0.10"
+    - "0.10.34"
+    - "0.10.35"
```

Comment:
I don t have one myself. Also, i think it is based on debian, which is really slow to update packages like node. So i was thinking maybe we should provide an all in one tarball.
We may need to integrate some users too.